.github/workflows/ci-postgres-mysql.yml

```diff
@@ -5,8 +5,6 @@ on:
     - cron: '0 0 * * *'
   workflow_dispatch:
   pull_request:
-    paths:
-      - packages/cli/src/databases/**
```

Comment:
Just temporary here to test the workflow.

Reply:
instead of removing this line, I think we should add `.github/workflows/ci-postgres-mysql.yml` to the list